### ISSUE TITLE
Crash on docstring with misindented character

### ISSUE THREAD

**Issue by JelleZijlstra** [psf/black#1585]
```
$ cat web/lib/webhelpers/html/blackbug.py 
def checkbox():
    """Create a check box.  Pass the empty string ("") to suppress the
    ID attribute entirely.

d
    The following HTML attributes may be set by keyword argument:
    """
$ ~/repos/black/.venv/bin/black  web/lib/webhelpers/html/blackbug.py 
error: cannot format web/lib/webhelpers/html/blackbug.py: INTERNAL ERROR: Black produced code that is not equivalent to the source.  Please report a bug on https://github.com/psf/black/issues.  This diff might be helpful: /tmp/blk_azbg7n69.log
Oh no! 💥 💔 💥
1 file failed to reformat.
$ ~/repos/black/.venv/bin/black --version
black, version 19.10b1.dev134+gb59a524.d20200811
$ cat /tmp/blk_azbg7n69.log
--- src
+++ dst
@@ -15,11 +15,11 @@
       body=
         Expr(
           value=
             Constant(
               value=
-                'Create a check box.  Pass the empty string ("") to suppress the\n ID attribute entirely.\n\nd\n The following HTML attributes may be set by keyword argument:',  # str
+                'Create a check box.  Pass the empty string ("") to suppress the\n ID attribute entirely.\n\n d\n The following HTML attributes may be set by keyword argument:',  # str
             )  # /Constant
         )  # /Expr
       decorator_list=
       name=
         'checkbox',  # str
```

I don't know why that random "d" was there but it shouldn't crash Black.

**Comment by hugovk** [comment]
Doesn't crash with last release 19.10b.

`git bisect` points to a4c11a75e12300abfbe4c36854e450d42bdd1ee7 from PR https://github.com/psf/black/pull/1053: "Re-indent the contents of docstrings".

**Comment by JelleZijlstra** [comment]
This is fixed on master, maybe by #1593.
